### PR TITLE
Adding Music section, adding color-identifiers mode, adding mac osx libraries, adding tramp-hdfs

### PR DESCRIPTION
**Pull request by tninja**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -213,6 +214,7 @@ Above, all enjoy using Emacs. The community, that more than anything, makes Emac
    - [[https://github.com/Malabarba/beacon][beacon]] - Never lose your cursor again.
    - [[https://github.com/gonewest818/dimmer.el][dimmer.el]] - Interactively highlight which buffer is active by dimming the others.
    - [[https://github.com/k-talo/volatile-highlights.el][volatile-highlights.el]] - Minor mode for visual feedback on some operations in Emacs.
+   - [[https://github.com/ankurdave/color-identifiers-mode][color-identifiers-mode]] - Color Identifiers is a minor mode for Emacs that highlights each source code identifier uniquely based on its name
```
Comment (agsdot) [inline README.org]: @tninja , please add a period after "based on its name."

File: README.org [hunk 2]
```diff
@@ -170,6 +170,7 @@ Above, all enjoy using Emacs. The community, that more than anything, makes Emac
    - [[https://github.com/escherdragon/sunrise-commander][Sunrise Commander]] - Twin-pane file manager for Emacs based on Dired and inspired by Midnight Commander.
    - [[https://github.com/Alexander-Miller/treemacs][Treemacs]] - a tree layout file explorer for Emacs.
    - [[https://github.com/sebastiencs/sidebar.el][Sidebar.el]] - A customizable file explorer with git integration for emacs.
+   - [[https://github.com/raghavgautam/tramp-hdfs][tramp-hdfs]] - Browser HDFS in Emacs with dired using Tramp
```
Comment (agsdot) [inline README.org]: @tninja make sure that lines end with a period: "dired using Tramp."

Also, the readme of @raghavgautam's project states "Browse HDFS(Hadoop Distributed File System) in Emacs using Tramp." That being said "Browser HDFS in Emacs using Tramp" sounds better as "Browse HDFS in Emacs using Tramp."

File: README.org [hunk 3]
```diff
@@ -609,6 +611,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
     - [[https://github.com/zk-phi/symon/][Symon]] - Tiny graphical system monitor.
     - [[https://github.com/jabranham/system-packages][system-packages]] - Manage your installed packages with emacs.
     - [[https://github.com/emacs-helm/helm-system-packages][Helm System Packages]] - A Helm interface to the package manager of your operating system.
+	- [[https://github.com/raghavgautam/osx-lib][osx-lib]] - Emacs functions for mac osx
```
Comment (agsdot) [inline README.org]: @tninja add a period at the end of the statement.

File: README.org [hunk 4]
```diff
@@ -609,6 +611,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
...
+	- [[https://github.com/benmaughan/spotlight.el][spotlight]] - Emacs package to query Mac OS X spotlight
```
Comment (agsdot) [inline README.org]: Capitalize spotlight and add a period .... "Spotlight."

File: README.org [hunk 5]
```diff
@@ -830,6 +834,12 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[https://github.com/johanvts/emacs-fireplace][fireplace]] - A cozy fireplace for emacs.
    - [[https://github.com/Fuco1/clippy.el][clippy]] - Show tooltip with function documentation at point.
 
+** Music
+
+   - [[https://github.com/mjago/Emacs/tree/master/lilypond][lilypond-mode]] - default emacs mode for [[http://lilypond.org/][lilypond]] editing
+   - [[https://github.com/nsceaux/lyqi][lyqi]] - 3rd party emacs mode for LilyPond editing
+   - [[https://github.com/jgkamat/alda-mode][alda-mode]] - emacs mode for [[https://github.com/alda-lang/alda][alda]], a music programming language for musicians written in java and clojure
+
```
Comment (agsdot) [inline README.org]: @tninja add periods to the end of each line.

**Comment by tninja** [comment]
Comments addressed. Thanks @agsdot

**Comment by agsdot** [comment]
@tninja thanks for putting in the PR and modifying the PR based on feedback. Your contribution to the Emacs community is greatly appreciated. Have a great week.

**Pull request merged**